code/modules/mob/living/carbon/carbon_movement.dm

```diff
@@ -1,6 +1,5 @@
 /mob/living/carbon/movement_delay()
```

Comment:
why not get rid of the whole proc since its just calling . = ..() now?

Reply:
there's still other things in the proc you fool

Reply:
I'm blind